src/CloudFoundry.Build.Tasks.IntegrationTests/Assets/testmanifest.yml

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Is the manifest supposed to be empty?
If so, can you rename to "emptymanifest"?